kubectl-pg/cmd/extVolume.go

```diff
@@ -0,0 +1,81 @@
+/*
+Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
please adjust the license as was described [in my commentary a week ago](https://github.com/zalando/postgres-operator/pull/579#issuecomment-505457247)

Reply:
@VineethReddy02